release/archives/linux/data-prepper-jdk-x64.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
nit: Consider using the safer alternative `#!/usr/bin/env bash`. The script will run using the instance of bash the current user has set instead of the system bash installation. Is rare cases, a user might have a newer version of bash installed.

Why change from sh to bash? When running Data Prepper in an Alpine or other minimal footprint container, bash is an additional dependency that will need to be installed.

Reply:
The script appears to require bash as it is. It did not work with the base images in the tar smoke tests.

Also, it used to be bash. I changed it to `sh` in #1682.